src/panels/lovelace/mixins/element-click-mixin.js

```diff
@@ -1,8 +1,12 @@
 import { dedupingMixin } from "@polymer/polymer/lib/utils/mixin.js";
+import { templateFactory } from "lit-html";
 import toggleEntity from "../common/entity/toggle-entity.js";
 import NavigateMixin from "../../../mixins/navigate-mixin";
 import EventsMixin from "../../../mixins/events-mixin.js";
 import computeStateName from "../../../common/entity/compute_state_name";
+import "@material/mwc-ripple";
+
+templateFactory.templateFactory = templateFactory;
```

Comment:
this is no longer needed if you update to latest version.

Reply:
Fixed